docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1221,6 +1203,456 @@ thor:
+        </example>
+
+        <para>This configuration enables:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Local inter-component communication with internal
+            certificates</para>
+          </listitem>
+
+          <listitem>
+            <para>Public-facing services with Let's Encrypt
+            certificates</para>
+          </listitem>
+
+          <listitem>
+            <para>Remote client certificates valid across multiple
+            domains</para>
+          </listitem>
+
+          <listitem>
+            <para>Code signing capabilities for ECL applications</para>
+          </listitem>
+        </itemizedlist>
+      </sect3>
+
+      <sect3 id="Troubleshooting">
+        <title>Troubleshooting Certificates</title>
+
+        <sect4 id="Certificate_Generation_Issues">
+          <title>Certificate Generation Issues</title>
+
+          <para>If certificates are not generating correctly with alternative
+          domains:</para>
+
+          <orderedlist>
+            <listitem>
+              <para>Verify that cert-manager is properly installed and
+              running</para>
+            </listitem>
+
+            <listitem>
+              <para>Check that all domains resolve properly via DNS</para>
+            </listitem>
+
+            <listitem>
+              <para>Examine Certificate and CertificateRequest resources for
+              error messages</para>
+            </listitem>
+
+            <listitem>
+              <para>Validate issuer configuration and permissions</para>
+            </listitem>
+          </orderedlist>
+        </sect4>
+
+        <sect4 id="Common_Error_Messages">
+          <title>Common Error Messages</title>
+
+          <para>Watch for these common issues when using alternative
+          domains:</para>
+
+          <variablelist>
+            <varlistentry>
+              <term>DNS validation failures</term>
+
+              <listitem>
+                <para>Ensure all alternative domains have proper DNS records
+                and are accessible from the certificate authority</para>
+              </listitem>
+            </varlistentry>
+
+            <varlistentry>
+              <term>Rate limit exceeded</term>
+
+              <listitem>
+                <para>Some certificate authorities impose limits on the number
+                of domains per certificate or certificates per
+                timeframe</para>
+              </listitem>
+            </varlistentry>
+
+            <varlistentry>
+              <term>Invalid domain format</term>
+
+              <listitem>
+                <para>Verify that all domains in alternativeDomains follow
+                proper DNS naming conventions</para>
+              </listitem>
+            </varlistentry>
+          </variablelist>
+        </sect4>
+      </sect3>
+
+      <sect3 id="Migration_Legacy_Configuration">
+        <title>Migration from Legacy Configuration</title>
```

Comment:
The real driver for this change was to provide a way to configure certs without using wildcards like `*.hpcc.internal`. Perhaps that could be mentioned in this list of migration improvements.